src/app/game/board/deck/deck.component.js

```diff
@@ -11,26 +11,39 @@ class Deck extends Component {
             drawPile: this.props.drawPile
         };
 
-        this.moveCard = this.moveCard.bind(this);
+    /*    this.moveCard = this.moveCard.bind(this);*/
+        this.moveCardDriver1 = this.moveCardDriver1.bind(this);
     }
     
     render() {
         return (
             <div className="deck-component">
                 <DiscardPile name="discardPile"
-                             cards={this.state.discardPile.cards} />
+                             discardPile={this.props.discardPile}
+                             cards={this.state.discardPile.cards}
+                             moveCardDriver1={this.moveCardDriver1}
+                />
                 <DrawPile name="drawPile"
+                          drawPile={this.props.drawPile}
                           cards={this.state.drawPile.cards}
-                          onCardClick={this.moveCard} />
+                          // onCardClick={this.moveCard}
+                          moveCardDriver1={this.moveCardDriver1}
+                />
             </div>
         );
     }
 
+    moveCardDriver1(card, sourcePile) {
```

Comment:
I don't understand this. what is card driver 0\1 ?

Reply:
we talked about it on the phone. and agreed to change the name if will keep it.